src/classes/GE_DefaultTemplate.cls

```diff
@@ -0,0 +1,234 @@
+                                                    '"defaultValue":null,"dataType":"STRING","dataImportFieldMappingDevNames":["Home_Zip_Postal_Code_f9807b51a"],"elementType":"field"},{"id"' + 
+                                                    ':"ac52960d-805e-fd42-542d-9b2b4a03c9e9","label":"Address: Mailing Country","customLabel":"Contact Country","required":false,"sectionId":' + 
+                                                    '"1cb17425-5ced-ebd6-cdaf-44556f648e3f","defaultValue":null,"dataType":"STRING","dataImportFieldMappingDevNames":["Home_Country_bb5c416c4"],' + 
+                                                    '"elementType":"field"}]},{"defaultDisplayMode":"expanded","displayRule":"displayRule","displayType":"accordion","elements":[{"customLabel":' + 
+                                                    '"Donation Date","dataImportFieldMappingDevNames":["Donation_Date_dfe3de83a"],"dataType":"DATE","elementType":"field","id":"ce4e9472-6c75-53d2-2e1c-36fcbcda210a"' + 
+                                                    ',"label":"Opportunity: Close Date","required":true,"sectionId":"3c42b004-03d7-90d5-98ed-e04f25af89e3"},{"customLabel":"Donation Amount",' + 
+                                                    '"dataImportFieldMappingDevNames":["Donation_Amount_91e74ad7f"],"dataType":"CURRENCY","elementType":"field","id":"ca2be0bf-d07e-2ec5-0bb6-6908e49dfe49"' + 
+                                                    ',"label":"Opportunity: Amount","required":true,"sectionId":"3c42b004-03d7-90d5-98ed-e04f25af89e3"},{"customLabel":"Payment Method","dataImportFieldMappingDevNames"' + 
+                                                    ':["Payment_Method_449599d68"],"dataType":"PICKLIST","elementType":"field","id":"149fb6d3-4a3a-c387-6d2c-1a77dcd14f7f","label":"Payment: Payment Method",' + 
+                                                    '"required":false,"sectionId":"3c42b004-03d7-90d5-98ed-e04f25af89e3"},{"customLabel":"Check/Reference Number","dataImportFieldMappingDevNames":' + 
+                                                    '["Payment_Check_Reference_Number_2947b94f7"],"dataType":"STRING","elementType":"field","id":"cc94d1ce-d93b-dc47-7703-cd89c27843ce","label":' + 
+                                                    '"Payment: Check/Reference Number","required":false,"sectionId":"3c42b004-03d7-90d5-98ed-e04f25af89e3"},{"componentName":"geFormWidgetAllocation"' + 
+                                                    ',"elementType":"widget","id":"7019830b-8026-5eac-b65e-0f74a3e5382f","label":"Allocations","required":false,"sectionId":"3c42b004-03d7-90d5-98ed-e04f25af89e3"}]' + 
+                                                    ',"id":"3c42b004-03d7-90d5-98ed-e04f25af89e3","label":"Donation Information"}]},"name":"Default Gift Entry Form"}';
```

Comment:
@KietTruongSFDO I didn't find anything for Primary Campaign Source in this. Does it need to be added, or did I just miss it?

Reply:
@melissabarber The Primary Campaign Source isn't there. It wasn't on the screenshots, should it be there?